contacts2/app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ContactsController < ApplicationController
+    def home
+    end
+    
+    def contact_params
```

Comment:
This works great! 👍 If you want to follow best practices, you can make this `contact_params` a private method. Not required in this case, but the idea is that this will prevent your params from being exposed to the outside world. Here is a good explanation on [stackoverflow](https://stackoverflow.com/questions/24561978/why-is-the-strong-parameters-method-private)